cmd/member-net-controller-manager/main.go

```diff
@@ -58,6 +60,9 @@ var (
 
 	tlsClientInsecure    = flag.Bool("tls-insecure", false, "Enable TLSClientConfig.Insecure property. Enabling this will make the connection inSecure (should be 'true' for testing purpose only.)")
 	fleetSystemNamespace = flag.String("fleet-system-namespace", "fleet-system", "The reserved system namespace used by fleet.")
+
+	isV1Alpha1APIEnabled = flag.Bool("enable-v1alpha1-apis", true, "If set, the agents will watch for the v1alpha1 APIs.")
+	isV1Beta1APIEnabled  = flag.Bool("enable-v1beta1-apis", false, "If set, the agents will watch for the v1beta1 APIs.")
```

Comment:
one minor comment, i'm thinking if we should change to enable-v1beta1-cluster-apis.

as it only enables the agents watch for the v1beta1 cluster apis and networking apis will still stay in v1alpha1.